integrations/salesforce/server.go

```diff
@@ -0,0 +1,42 @@
+package salesforce
+
+import (
+	"net/http"
+
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/muxes"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/web/static"
+)
+
+// Start initializes all the HTTP handlers of all the Salesforce integrations. This
+// includes connection UIs, connection initialization webhooks, and event webhooks.
```

Comment:
Recent change in #1067
```suggestion
// Start initializes all the HTTP handlers of the integration.
// This includes an internal connection UI, webhooks for AutoKitteh
// connection initialization, and asynchronous event webhooks.
```